user-guide/Basic_Functionality/Visio/linking_shapes/Linking_a_shape_to_a_video_thumbnail.md

```diff
@@ -199,6 +199,21 @@ const video_url_via_proxy = `${proxy_url}/${btoa(video_url)}${file_extension}`;
 const videothumbnails = `https://dma-server/videothumbnails/video.htm?type=HTML5-HLS&source=${encodeURIComponent(video_url_via_proxy)}`; // encodeURIComponent = url-encode
 ```
 
+#### Using the HLS Proxy server behind a front-end web server
+
+Optionally, it is possible to run the HLS Proxy server behind a front-end web server with a [reverse proxy](xref:Dashboard_Gateway_installation#reverse-proxy). The `--host` parameter can be specified containing the public FQDN and port of the public-facing server. This way, no separate port needs to be made publicly accessible, and there is no need to configure the HTTPS certificate on the HLS proxy because the web server can take care of that.
+
+Any leading path (for example `/hls-proxy`) in the request is echoed into all proxied URLs in the modified HLS manifest by the HLS Proxy.
```

Comment:
This sentence is a bit hard to understand. Do you mean this?

```suggestion
The HLS Proxy echoes any leading path (for example `/hls-proxy`) in the request into all proxied URLs in the modified HLS manifest.
```

Reply:
Yes indeed, that's correct